lib/__tests__/parse-numeric.test.js

```diff
@@ -3,71 +3,44 @@ const validNumbers = require('./__fixtures__/valid-numbers')
 const validNumericStrings = require('./__fixtures__/valid-numeric-strings')
 const invalidNumericStrings = require('./__fixtures__/invalid-numeric-strings')
 
-describe('module parse-numeric', () => {
-  describe('function parseNumeric (value: any)', () => {
-    describe('when the "value" parameter is', () => {
-      describe('a {string} that represents a {number}', () => {
-        it('will convert the {string} to a {number}', () => {
-          validNumericStrings.forEach(value => {
-            const number = parseNumeric(value)
-            expect(typeof number).toBe('number')
-          })
-        })
-      })
-
-      describe('a non-numeric {string}', () => {
-        it('will return the original value', () => {
-          invalidNumericStrings.forEach(value => {
-            expect(parseNumeric(value)).toBe(value)
-          })
-        })
-      })
-
-      describe('a {number}', () => {
-        it('will return the {number}', () => {
-          validNumbers.forEach(num => {
-            const number = parseNumeric(num)
-            expect(typeof number).toBe('number')
-            expect(number).toEqual(num)
-          })
-        })
+describe('parseNumeric', () => {
+  describe('when the "value" parameter is', () => {
+    const isTypeOfNumber = (val) => typeof val === 'number'
+    const parseValues = (values) => values.map(parseNumeric)
```

Comment:
**This source code conforms to the [JavaScript Standard Code Style](https://standardjs.com/rules.html#semicolons)**. It is flagged as a violation/code smell/warning due to misconfigured (or simply neglected) settings.

I have since configured settings properly, and marking this as "resolved."